automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala

```diff
@@ -45,48 +42,43 @@ case class Table(private val id: String)(implicit webDriver: WebDriver)
   }
 
   def goToPreviousPage(): Unit = {
-    click on prevPageButton
+    prevPageButton.doClick()
     awaitReady()
   }
 
   def goToNextPage(): Unit = {
-    click on nextPageButton
+    nextPageButton.doClick()
     awaitReady()
   }
 
   def goToPage(page: Int): Unit = {
-    click on pageButton(page)
+    pageButton(page).doClick()
     awaitReady()
   }
 
   def selectPerPage(perPage: Int): Unit = {
-    singleSel(perPageSelector).value = perPage.toString
+    perPageSelector.select(perPage.toString)
     awaitReady()
   }
 
   def getData: List[List[String]] = {
     import scala.collection.JavaConversions._
-
-    awaitReady()
     val rows = tableBody.webElement.findElements(By.cssSelector("[data-test-class='table-row']")).toList
     rows.map(_.findElements(By.cssSelector("[data-test-class='table-cell']")).toList.map(_.getText))
   }
 
   def readColumnHeaders: List[String] = {
-    awaitReady()
     readAllText(columnHeaders)
   }
 
-  def clearFilter: Unit = {
-    searchField(filterField).value = ""
-    click on filterButton
```

Comment:
I'm not convinced that we don't need to click the filter button and wait. I know that Ursa ran into some timing issues with this that adding these two lines fixed. It's possible that it worked locally but failed when running headless from Jenkins.